euler.py

```diff
@@ -0,0 +1,49 @@
+import time 
+
+from numba import jit
+
+@jit
```

Comment:
@njit betters suites here  [https://numba.readthedocs.io/en/stable/user/5minguide.html#other-things-of-interest](https://numba.readthedocs.io/en/stable/user/5minguide.html#other-things-of-interest)